### PR TITLE
Add NTP server support

### PR DESCRIPTION
**Pull request by gsanchietti**
NethServer/dev#5249

### CODE REVIEW

File: root/etc/e-smith/events/actions/nethserver-dc-fixchroot [hunk 1]
```diff
@@ -28,7 +28,21 @@ fi
...
+    fi
+done
+
+# Enable ntpd service
+if ! systemctl --quiet -M nsdc is-active ntpd; then
+    systemctl -M nsdc enable ntpd
 fi
+
+# Set ntpd permissions
+uid=$(grep ntp ${nsroot}/etc/passwd | awk -F: '{print $3}')
```
Comment (DavidePrincipi) [inline root/etc/e-smith/events/actions/nethserver-dc-fixchroot:45]: Expanded variable not actually used!

**Review by DavidePrincipi — CHANGES_REQUESTED** [review]
Unused ``uid`` variable

**Comment by DavidePrincipi** [comment]
BTW Why not use chronyd also for container? Does ntpd offer some special feature for MS clients?

File: root/etc/e-smith/events/actions/nethserver-dc-fixchroot (hunk 1 above)
Comment (gsanchietti) [reply root/etc/e-smith/events/actions/nethserver-dc-fixchroot:45]: Fixed.

**Comment by gsanchietti** [comment]
> BTW Why not use chronyd also for container? Does ntpd offer some special feature for MS clients?

Yes, ntp signed protocol.

See https://wiki.samba.org/index.php/Time_Synchronisation

**Review by DavidePrincipi — APPROVED** [review]

**Pull request merged**